src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Public.Data/DataSetVersionService.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services.Interfaces.Public.Data;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database;
+using Microsoft.EntityFrameworkCore;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Services.Public.Data;
+
+public class DataSetVersionService(
+    ContentDbContext contentDbContext,
+    PublicDataDbContext publicDataDbContext) 
+    : IDataSetVersionService
+{
+    public async Task<List<DataSetVersionStatusSummary>> GetStatusesForReleaseVersion(Guid releaseVersionId)
+    {
+        var releaseFileIds = await contentDbContext
+            .ReleaseFiles
+            .AsNoTracking()
+            .Where(rf => rf.ReleaseVersionId == releaseVersionId && rf.File.Type == FileType.Data)
+            .Select(rf => rf.Id)
+            .ToListAsync();
```

Comment:
I don't think it hurts to have `AsNoTracking` here but the result doesn't contain any entity types so no tracking would be done anyway.

Reply:
Good spot - I've removed thanks - originally I think I had this returning entities but I can't remember!